src/database/cassandra-handler/init-scripts/19__bench_mark_data_reports.cql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE reports_summary ADD weights_data text;
```

Comment:
maybe changed weights_data to benchmark_weights_data

Reply:
Fixed